partials/expenses_input.html

```diff
@@ -14,7 +16,7 @@
                     class="steps"
                     min="0"
                     max="99999"
-                    ng-model="expenses_ctrl.expenses" />
+                    ng-model="expenses" />

```

Comment:
Do you mean just make the value a property of an object? Like, in the controller, it'd be defined as $scope.userData.expenses ? The reason I had as part of an object before ( expneses_ctrl.expenses) was to avoid overusing $scope. But, I had to change it back because I needed to add a $watch to the value so that even when the user advances in the flow without submitting, the value is stored in UserDataCache. So, I could just use $scope.someObject.value and solve both issues.